code/modules/mob/living/silicon/robot/robot_modules/station.dm

```diff
@@ -884,6 +884,11 @@ var/global/list/robot_modules = list(
 	P.synths = list(plastic)
 	src.modules += P
 
+/obj/item/weapon/robot_module/drone/New(mob/living/silicon/robot/robot)
```

Comment:
unnecessary proc override get rid of this and put the code for adding those two things to mdoules into the main New() code.
we only do double proc overrides on the same path when really necessary for modularization or other purposes, it imapcts performance and readability.